lang/cs/Org.Apache.REEF.Client/AzureBatch/AzureBatchDotNetClient.cs

```diff
@@ -45,6 +47,12 @@ public sealed class AzureBatchDotNetClient : IREEFClient
         private readonly AzureBatchService _batchService;
         private readonly JobJarMaker _jobJarMaker;
         private readonly AzureBatchFileNames _azbatchFileNames;
+        private readonly string _azureBatchAccountName;
```

Comment:
Definitely we can. I am kind not sure which one is better practice. Having a wrapper only contains getters with no logic, or just like this. Since we are using injection here, no pain to call new Instance(param1, param2, .....)

Let's wait for another vote maybe?

Reply:
Discussed offline. I will fix this by implementing helper methods in AzureBatchService class which holds the credentials. So we don't need pass them around.